specification/monitor/resource-manager/readme.md

```diff
@@ -35,11 +35,10 @@ title: MonitorManagementClient
 ``` yaml
 description: Monitor Management Client
 openapi-type: arm
-tag: package-2022-06
-
+tag: package-2022-06-15-only
```

Comment:
Hi @avjerafi , I don't find "package-2022-06-15-only" defined in readme.md, this should be the reason that make SDK generations failed.

Reply:
@changlong-liu You're right. Thanks! Pushed a fix.